packages/contracts/contracts/L1/ValidatorManager.sol

```diff
@@ -600,11 +602,11 @@ contract ValidatorManager is ISemver, IValidatorManager {
      */
     function _updatePriorityValidator() private {
         uint120 weightSum = activatedValidatorTotalWeight();
+        uint256 nextFinalizeOutputIndex = L2_ORACLE.nextFinalizeOutputIndex();
 
-        if (weightSum > 0) {
-            uint256 latestFinalizedOutputIndex = L2_ORACLE.latestFinalizedOutputIndex();
+        if (weightSum > 0 && nextFinalizeOutputIndex > 0) {
```

Comment:
Question about this: Is this addition for the test codes?

Reply:
No, it's because we need to select next priority validator based on finalized output.